[2024-03-19T17:03:19.649+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: recalls_etl_v1.get_raw_data manual__2024-03-19T17:03:17.964817+00:00 [queued]>
[2024-03-19T17:03:19.653+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: recalls_etl_v1.get_raw_data manual__2024-03-19T17:03:17.964817+00:00 [queued]>
[2024-03-19T17:03:19.653+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 1
[2024-03-19T17:03:19.659+0000] {taskinstance.py:2217} INFO - Executing <Task(PythonOperator): get_raw_data> on 2024-03-19 17:03:17.964817+00:00
[2024-03-19T17:03:19.668+0000] {standard_task_runner.py:60} INFO - Started process 60 to run task
[2024-03-19T17:03:19.671+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'recalls_etl_v1', 'get_raw_data', 'manual__2024-03-19T17:03:17.964817+00:00', '--job-id', '29', '--raw', '--subdir', 'DAGS_FOLDER/recalls_etl_v1.py', '--cfg-path', '/tmp/tmpqt7e_bc0']
[2024-03-19T17:03:19.672+0000] {standard_task_runner.py:88} INFO - Job 29: Subtask get_raw_data
[2024-03-19T17:03:19.724+0000] {task_command.py:423} INFO - Running <TaskInstance: recalls_etl_v1.get_raw_data manual__2024-03-19T17:03:17.964817+00:00 [running]> on host b1b6e0a46fce
[2024-03-19T17:03:19.787+0000] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='recalls_etl_v1' AIRFLOW_CTX_TASK_ID='get_raw_data' AIRFLOW_CTX_EXECUTION_DATE='2024-03-19T17:03:17.964817+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-03-19T17:03:17.964817+00:00'
[2024-03-19T17:03:19.789+0000] {taskinstance.py:2731} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 444, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 414, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 200, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 217, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/recalls_etl_v1.py", line 38, in get_recall_data
    response = requests.get(url)
  File "/home/airflow/.local/lib/python3.8/site-packages/requests/api.py", line 73, in get
    return request("get", url, params=params, **kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/requests/api.py", line 59, in request
    return session.request(method=method, url=url, **kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/requests/sessions.py", line 589, in request
    resp = self.send(prep, **send_kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/requests/sessions.py", line 697, in send
    adapter = self.get_adapter(url=request.url)
  File "/home/airflow/.local/lib/python3.8/site-packages/requests/sessions.py", line 794, in get_adapter
    raise InvalidSchema(f"No connection adapters were found for {url!r}")
requests.exceptions.InvalidSchema: No connection adapters were found for '[{\'NID\': \'73357\', \'Title\': \'"Unauthorized sexual enhancement products may pose serious health risks"\', \'URL\': \'https://recalls-rappels.canada.ca/en/alert-recall/unauthorized-sexual-enhancement-products-may-pose-serious-health-risks\', \'Organization\': \'Health Canada\', \'Product\': \'Various unauthorized health products promoted for sexual enhancement\', \'Issue\': \'Unauthorized product\', \'What you should do\': \'"Stop using the products listed in the table. Consult your health care professional if you have used these products and have health concerns, and for advice on which health products are best for you and your family.Read product labels to verify that health products have been authorized for sale by Health Canada. Authorized health products have an eight-digit Drug Identification Number (DIN), Natural Product Number (NPN) or Homeopathic Drug Number (DIN-HM). You can also check if products have been authorized for sale by searching Health Canada\\\'s Drug Product Database and Licensed Natural Health Product Database.Report any health product adverse events or complaints to Health Canada.See these helpful links for additional information on buying health products safely:Buying drugs over the InternetAdulteration of natural health productsRisks of buying natural health products onlineThe safe use of health products for weight lossWhat companies should know:Selling unauthorized health products in Canada is illegal.When Health Canada identifies unauthorized products that may pose serious health risks, Health Canada takes appropriate action to prevent further distribution and informs the public. This includes working with the Canada Border Services Agency to help prevent further importation of unauthorized products."\', \'Category\': \'Health products - Drugs\', \'Recall class\': \'\', \'Last updated\': \'2024-01-29\', \'Archived\': \'No\'}, {\'NID\': \'74816\', \'Title\': \'"Global Medium"\', \'URL\': \'https://recalls-rappels.canada.ca/en/alert-recall/global-medium\', \'Organization\': \'Health Canada\', \'Product\': \'Global Medium\', \'Issue\': \'Performance issue\', \'What you should do\': \'\', \'Category\': \'Health products - Medical devices - Chemistry\', \'Recall class\': \'Type II\', \'Last updated\': \'2023-12-27\', \'Archived\': \'No\'}]'
[2024-03-19T17:03:19.794+0000] {taskinstance.py:1149} INFO - Marking task as FAILED. dag_id=recalls_etl_v1, task_id=get_raw_data, execution_date=20240319T170317, start_date=20240319T170319, end_date=20240319T170319
[2024-03-19T17:03:19.808+0000] {standard_task_runner.py:107} ERROR - Failed to execute job 29 for task get_raw_data (No connection adapters were found for '[{\'NID\': \'73357\', \'Title\': \'"Unauthorized sexual enhancement products may pose serious health risks"\', \'URL\': \'https://recalls-rappels.canada.ca/en/alert-recall/unauthorized-sexual-enhancement-products-may-pose-serious-health-risks\', \'Organization\': \'Health Canada\', \'Product\': \'Various unauthorized health products promoted for sexual enhancement\', \'Issue\': \'Unauthorized product\', \'What you should do\': \'"Stop using the products listed in the table. Consult your health care professional if you have used these products and have health concerns, and for advice on which health products are best for you and your family.Read product labels to verify that health products have been authorized for sale by Health Canada. Authorized health products have an eight-digit Drug Identification Number (DIN), Natural Product Number (NPN) or Homeopathic Drug Number (DIN-HM). You can also check if products have been authorized for sale by searching Health Canada\\\'s Drug Product Database and Licensed Natural Health Product Database.Report any health product adverse events or complaints to Health Canada.See these helpful links for additional information on buying health products safely:Buying drugs over the InternetAdulteration of natural health productsRisks of buying natural health products onlineThe safe use of health products for weight lossWhat companies should know:Selling unauthorized health products in Canada is illegal.When Health Canada identifies unauthorized products that may pose serious health risks, Health Canada takes appropriate action to prevent further distribution and informs the public. This includes working with the Canada Border Services Agency to help prevent further importation of unauthorized products."\', \'Category\': \'Health products - Drugs\', \'Recall class\': \'\', \'Last updated\': \'2024-01-29\', \'Archived\': \'No\'}, {\'NID\': \'74816\', \'Title\': \'"Global Medium"\', \'URL\': \'https://recalls-rappels.canada.ca/en/alert-recall/global-medium\', \'Organization\': \'Health Canada\', \'Product\': \'Global Medium\', \'Issue\': \'Performance issue\', \'What you should do\': \'\', \'Category\': \'Health products - Medical devices - Chemistry\', \'Recall class\': \'Type II\', \'Last updated\': \'2023-12-27\', \'Archived\': \'No\'}]'; 60)
[2024-03-19T17:03:19.853+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 1
[2024-03-19T17:03:19.872+0000] {taskinstance.py:3312} INFO - 0 downstream tasks scheduled from follow-on schedule check
